src/QsCompiler/Transformations/CallGraphWalker.cs

```diff
@@ -0,0 +1,319 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Looks like there are still a bunch of todo's left throughout this file. What should I look at exactly, and what isn't ready for review yet?

Reply:
I rewrote some of the ToDo's to make them clearer. If there is one or two in particular that you need more explanation, on feel to post a comment 😊.